src/widgets/TOCWidget.vue

```diff
@@ -18,28 +19,39 @@
     },
     watch: {
       $route: 'fetchData',
+      defaultTocUrn: 'fetchData',
     },
     data() {
       return {
         toc: null,
       };
     },
     computed: {
+      metadata() {
+        return this.$store.getters[`${WIDGETS_NS}/metadata`];
+      },
+      defaultTocUrn() {
+        return this.metadata && this.metadata.defaultTocUrn
+          ? this.metadata.defaultTocUrn
+          : this.rootTocUrn;
+      },
       endpoint() {
-        return 'https://mini-stack-a-feature-se-j47yu0.herokuapp.com';
+        return this.$store.state[`${WIDGETS_NS}`].tocEndpoint;
```

Comment:
Probably want to define a getter (like `this.$store.getters[`${WIDGETS_NS}/metadata`]` above) for this, just to be consistent with the other widgets

Reply:
Ah ok nevermind, looks like you're handling this value in a different way than a normal state value so this is probably moot.